Tests/com/drew/imaging/jpeg/JpegMetadataReaderTest.java

```diff
@@ -57,6 +59,24 @@ public void testExtractXmpMetadata() throws Exception
         assertNotNull(directory);
     }
 
+    @Test
+    public void testFStopFormatting() throws Exception
+    {
+        Locale previousLocale = Locale.getDefault();
+
+        // Setting this locale would cause an f-number with a comma:
+        Locale.setDefault(Locale.forLanguageTag("nl-NL"));
+
+        try {
+            Metadata metadata = JpegMetadataReader.readMetadata(new File("Tests/Data/withExif.jpg"));
+            final ExifSubIFDDirectory subIFDDirectory = metadata.getFirstDirectoryOfType(ExifSubIFDDirectory.class);
+            assertEquals("f/4.0", subIFDDirectory.getDescription(ExifDirectoryBase.TAG_FNUMBER));
```

Comment:
Ok sorry I think I understand the PR now. Are you saying that in `nl-NL` the output should be `f/4.0` not `f/4,0`? Is this really what all locales that use a comma as a decimal separator would expect?